lib/fastlane/plugin/settings_bundle/helper/settings_bundle_helper.rb

```diff
@@ -139,8 +139,8 @@ def xcodeproj_path_from_params(params)
           repo_path = Bundler.root
 
           all_xcodeproj_paths = Dir[File.expand_path(File.join(repo_path, '**/*.xcodeproj'))]
-          # find an xcodeproj (ignoring the Cocoapods one)
-          xcodeproj_paths = Fastlane::Actions.ignore_cocoapods_path(all_xcodeproj_paths)
+          # find an xcodeproj (ignoring dependencies)
+          xcodeproj_paths = Fastlane::Helper::XcodeprojHelper.find(all_xcodeproj_paths)
```

Comment:
You can directly pass in `repo_path` to `XcodeprojHelper.find`. Passing in an array won't work, unfortunately. See #27.